component-library/src/app/components/component-preview/component-preview.component.scss

```diff
@@ -22,15 +22,15 @@ $selectors: "app-component-preview";
 
   .preview-wrapper {
     box-sizing: border-box;
-    padding: 59px 65px 44px 65px;
```

Comment:
Mahsa just changed this yesterday for the input documentation component preview. Are we leaving it as 0 64px and then overriding it in the docsite scss for specific paddings, etc?

Reply:
Please refer to https://www.figma.com/file/TPwx1HcbXRCZeIDePI4Y4N/Documentation-Site?type=design&node-id=6640-193516&mode=dev for the updated design.

Reply:
Okay, so you've made the changes to the component to adjust for mobile viewports and we can remove any styling in the input and icon button page since you've captured it here.

Thanks